website/www/site/content/en/documentation/ml/side-input-updates.md

```diff
@@ -128,5 +121,15 @@ class PostProcessor(beam.DoFn):
     predicted_class_name = imagenet_labels[predicted_class]
     return predicted_class_name.title(), element.model_id
 
-(inference_pcoll | "PostProcessor" >> PostProcessor())
+post_processor_pcoll = (inference_pcoll | "PostProcessor" >> PostProcessor())
 ```
+
+### Run the pipeline
+```python
+result = pipeline.run().wait_until_finish()
+```
+Once the pipeline is run with initial settings, upload a model matching the `file_pattern` to GCS bucket. After some time, you will see that your pipeline starts to use the updated model instead of the initial model. 
```

Comment:
```suggestion
After you run the pipeline with the initial settings, upload a model matching the `file_pattern` to the Google Cloud Storage bucket. Your pipeline will use the updated model instead of the initial model. 
```